packages/@aws-cdk/aws-pipes-sources-alpha/test/integ.kinesis.ts

```diff
@@ -0,0 +1,69 @@
+import { randomUUID } from 'crypto';
+import { ITarget, Pipe, TargetConfig } from '@aws-cdk/aws-pipes-alpha';
+import { ExpectedResult, IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import { KinesisSource, KinesisStartingPosition, OnPartialBatchItemFailure } from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-pipes-sources-kinesis');
+const sourceKinesisStream = new cdk.aws_kinesis.Stream(stack, 'SourceKinesisStream');
+const targetQueue = new cdk.aws_sqs.Queue(stack, 'TargetQueue');
+
+class TestTarget implements ITarget {
+  targetArn: string;
+
+  constructor(private readonly queue: cdk.aws_sqs.Queue) {
+    this.queue = queue;
+    this.targetArn = queue.queueArn;
+  }
+
+  bind(_pipe: Pipe): TargetConfig {
+    return {
+      targetParameters: {},
+    };
+  }
+
+  grantPush(pipeRole: cdk.aws_iam.IRole): void {
+    this.queue.grantSendMessages(pipeRole);
+  }
+}
+
+const sourceUnderTest = new KinesisSource(sourceKinesisStream, {
+  batchSize: 1,
+  maximumBatchingWindow: cdk.Duration.seconds(20),
+  maximumRecordAge: cdk.Duration.seconds(60),
+  maximumRetryAttempts: 3,
+  onPartialBatchItemFailure: OnPartialBatchItemFailure.AUTOMATIC_BISECT,
+  parallelizationFactor: 1,
+  startingPosition: KinesisStartingPosition.LATEST,
+});
```

Comment:
@nmussy I couldn't get this to deploy with a dead-letter target either. But given this is `alpha`, and you can't even specify dead-letter in the Console, I think we're good here. I did add unit tests to make sure the permissions get added.

Reply:
Might be a good idea to report the issue and not to expose the Kinesis DLQ parameter to the user in the meantime then

Reply:
@pahud Is this something you can raise with the CFN/Pipes team? When you create a Kinesis source with a dead-letter config, it doesn't appear to stabilize in time.